### ISSUE TITLE
how can I log error in cobrand pm files ?

### ISSUE THREAD

**Issue by loran750** [mysociety/fixmystreet#1404]
hello

i would like to print or to log error to the console in /var/www/domain.com/fixmystreet/perllib/FixMyStreet/Cobrand files but I can't output any data.

Tried some functions but without success in those files : 

debug_print('Test 1');
warn sprintf 'Test 2';
$c->log->debug( 'Test3' );

regards

**Comment by dracos** [comment]
This depends how you're running your server, but here at least, $c->log->inf

**Comment by dracos** [comment]
Whoops, hit go too early. This depends how you're running your server, but here at least, `$c->log->info` or `debug` appear to work okay. `debug` is turned off by default unless you're running in debug mode.

I added a `$self->{c}->log->info("HELLO")` line and a `warn "GOODBYE"` line to my cobrand's `path_to_web_templates` and got:

```
$ script/fixmystreet_app_server.pl --fork --restart
2016/05/17-09:41:18 Starman::Server (type Net::Server::PreFork) starting! pid(43116)
Resolved [*]:3000 to [0.0.0.0]:3000, IPv4
Binding to TCP port 3000 on host 0.0.0.0 with IPv4
Setting gid to "20 20 20 12 61 79 80 81 98 402 33 100 204 395 398 399"
Starman: Accepting connections at http://*:3000/
```
Made a request to http://localhost:3000/ and got:
```
GOODBYE at /Users/matthew/Projects/mySociety/FixMyStreet/fixmystreet/script/../perllib/FixMyStreet/Cobrand/Foo.pm line 10, <DATA> line 1003.
[info] HELLO
```

So they both log there okay. In an Apache set up, `$c->log` lines appear to be logged to the correct error log okay, with `STDERR` output going to the main Apache error log. Not sure what `debug_print` is.

Hope that's helpful.

**Comment by loran750** [comment]
hello dracos

it works like a charm. Thank you : 

GOODBYE at /var/www/xxxxx/fixmystreet/script/../perllib/FixMyStreet/Cobrand/xxxxxx.pm line 29, <DATA> line 1003.
[info] HELLO

regards
